test/test_browser.py

```diff
@@ -5329,6 +5329,12 @@ def test_wasm_worker_lock_wait2(self):
   def test_wasm_worker_lock_async_acquire(self):
     self.btest_exit('wasm_worker/lock_async_acquire.c', cflags=['--closure=1', '-sWASM_WORKERS'])
 
+  # Tests emscripten_lock_async_acquire() function when lock is acquired both synchronously and asynchronously.
+  @also_with_minimal_runtime
+  @flaky('https://github.com/emscripten-core/emscripten/issues/25270')
```

Comment:
Because this test doesn't actually create any workers this flaky decorator isn't applicable is it?